pulp_container/app/viewsets.py

```diff
@@ -839,7 +839,7 @@ def copy_manifests(self, request, pk):
         responses={202: AsyncOperationResponseSerializer},
         request=serializers.OCIBuildImageSerializer,
```

Comment:
Can you check whether this can be omitted now? Spectacular should derive it from the action.

Reply:
According to the documentation (https://drf-spectacular.readthedocs.io/en/latest/customization.html?highlight=discover#step-2-extend-schema), we can leave the parameter `request` empty: ` you only need to override what was not properly discovered in the introspection`.

Reply:
The serializer is correctly detected in the REST documentation too.